src/Utility/TriangleQuadrature.C

```diff
@@ -42,7 +42,7 @@ static const double T4[8] = {
 };
 //! \brief 4-point rule weights.
 static const double W4[4] = {
- -0.5635,
+ -0.5625,
```

Comment:
Most likely you're right, but double check with Tom Hughes' FEM book where these are taken from before approval.

Reply:
yeah, i already did. also the sanity check should make it clear - the weights should sum to 1. they don't do currently, they do after the fix.

Reply:
btw, in cause you wonder, the reason i ran into this is that coverage is now part of the static analysis jobs, and i noticed this class had no tests.